### PR TITLE
feat(FEC-10695): add screen lock orientation to studio

### PR DESCRIPTION
**Pull request by Yuvalke**

### CODE REVIEW

**Comment by OrenMe** [comment]
@Yuvalke please add an image showing the new section

**Comment by Yuvalke** [comment]
Screen Shot 2021-01-25 at 15 36 56
@OrenMe

**Review by OrenMe — APPROVED** [review]

**Comment by OrenMe** [comment]
@Yuvalke can you also add with drop down open. Thanks

File: app/js/services/v3Properties.json [hunk 1]
```diff
@@ -162,6 +162,50 @@
         "model": "config.playback.allowMutedAutoPlay",
         "endline": "true"
       },
+      {
+        "label": "Screen lock orientation",
+        "helpnote": "Select the default screen lock orientation for the player",
```
Comment (OrenMe) [inline app/js/services/v3Properties.json:167]:
```suggestion
        "helpnote": "Select the default screen lock orientation for the player when in fullscreen",
```

**Review by RoyBregman — APPROVED** [review]

**Pull request merged**